tests/common_docker.py

```diff
@@ -77,6 +77,9 @@ def docker_compose_cmd(arg_list, use_common_files=True, env=None):
                 if "up -d" in arg_list:
                     store_logs()
 
+                # Return as string (Python 2/3 compatible)
+                if isinstance(output, bytes):
+                    return output.decode("utf-8")
```

Comment:
Minor, mostly discussing code style: The default `decode()` is UTF-8. Normally I'd go for being explicit, but as this tends to be used *a lot* in Python 3, I would suggest using the shortest possible variant. What do you think?

Reply:
Agreed :+1: